bliss/models/galsim_encoder.py

```diff
@@ -190,7 +190,7 @@ def encode(self, image_ptiles: Tensor, tile_locs: Tensor) -> Tensor:
             ms=max_sources,
         )
 
-    def sample(self, image_ptiles: Tensor, tile_locs: Tensor):
+    def sample(self, image_ptiles: Tensor, tile_locs: Tensor, deterministic: Optional[bool]):
```

Comment:
why did you add this `deterministic` flag?

Reply:
This was just to match the other GalaxyEncoder (since we support both in encoder.py)